### ISSUE TITLE
Is there any new work ongoing on this project?

### ISSUE THREAD

**Issue by robertgoolsby** [jpchip/baby-connect-nightmare#1]
I'd like to setup a dashboard with many of the data points but having troubles with the latest versions of dependencies.

**Comment by jpchip** [comment]
No, sorry, haven't worked on this in a long time. I believe a few people have forked it though and added new features and fixes, might want to check those out.

**Comment by robertgoolsby** [comment]
No worries. I think I got what I needed at least so far.  I'll update below later on with a few of the changes that I had to make in the event someone comes finds this post.

**Comment by jpchip** [comment]
Cool. If you want you can make a pull request, I'll merge your changes back in at least.
